src/app/workflow/workflow.component.html

```diff
@@ -90,11 +90,11 @@
               novalidate>
           <button type="submit"
                   class="btn btn-link pull-right"
-                  [disabled]="!labelForm.form.valid"
+                  [disabled]="!labelForm.form.valid && !isWorkflowPublic"
```

Comment:
container.component.html has different additions compared to this workflow.component.html.  The disabled expression doesn't seem quite right.